shopinvader_product_variant_selector/readme/CREDITS.rst

```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
+
+* Abilis
```

Comment:
can we say Cosanum as well? :)

Reply:
yes I am going to fix this and other comment